packages/opentelemetry-core/src/platform/browser/log-level.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @returns the default log level for opentelemetry.
+ */
+import { LogLevel } from '../../common/types';
+
+export function getLogLevel(): LogLevel {
```

Comment:
Log level is always info for browser? @obecny what is the typical way to configure things like this

Reply:
By default info, warning and error are always shown so this will be fine and we will have the same behaviour for node and browser. 
I'm wondering a different thing - if we could implement the same mechanism and check global "OTEL_LOG_LEVEL" on `window` so mimic the same behaviour. But this might be something we can discuss on Wednesday perhaps. If we allow "globals" to have impact on otel then maybe we should have the same using window. There might be some cons with this but this is just an idea and not in this PR. Maybe @draffensperger can add some input here on the idea to mimic env variables as globals on `window` ?

Reply:
Do we still allow the parent trace ID and span ID to be set via a window variable from the server? If so, there is some precedent of allowing configuration via a window var.

The nice thing about making it available to be set on the window is that it allows a workflow where we build the OTel library and someone gets it as a `<script>` tag via a CDN and then adds configuration to it via a separate script tag, sort of like this https://github.com/census-instrumentation/opencensus-web#using-via-a-script-tag where I set the sampling rate based on window vars.

Reply:
I think if we do that, we should have some prefix for OTel config settings for the variables on the window like `_otel` or something like that to avoid conflicting with someone use of variables of the same name on window.